lib/cog/models/ecto_json.ex

```diff
@@ -52,6 +52,37 @@ defmodule Cog.Models.EctoJson do
     end
   end
 
+  def render_permissions(model, opts \\ []) do
+    opts = Keyword.merge(@render_defaults, opts)
+
+    policy = Keyword.fetch!(opts, :policy)
+    data = do_render(model, policy)
+    |> format_permissions(model)
+
+    case Keyword.fetch!(opts, :envelope) do
+      nil ->
+        data
+      envelope ->
+        Map.put(%{}, envelope, data)
+    end
+  end
```

Comment:
:+1: I will back out these changes and start with using phoenix views for the permissions changes.